lib/http_server.ex

```diff
@@ -1,4 +1,7 @@
 defmodule HTTPServer do
```

Comment:
This module is getting a little long - wondering if everything in here belongs in here or if any of the methods could be extracted into their own module(s)? Thoughts?

Reply:
Same [thought](https://github.com/kelseyroy/http_server/pull/3#discussion_r1213342021), at the same time

Reply:
![image](https://github.com/kelseyroy/http_server/assets/26009042/b886673e-3a1f-47ac-80e6-0f32db7172d2)

Reply:
Totally agree about needing to break down this module. I was thinking of adding some of these formatting/parsing functions to the existing request/response (depending on which one they deal with).